ompi/mca/coll/ucc/coll_ucc_module.c

```diff
@@ -175,10 +175,14 @@ static ucc_status_t oob_allgather_test(void *req)
         senddatafrom = (rank - oob_req->iter + size) % size;
         tmprecv = (char*)oob_req->rbuf + (ptrdiff_t)recvdatafrom * (ptrdiff_t)msglen;
         tmpsend = (char*)oob_req->rbuf + (ptrdiff_t)senddatafrom * (ptrdiff_t)msglen;
-        MCA_PML_CALL(isend(tmpsend, msglen, MPI_BYTE, sendto, MCA_COLL_BASE_TAG_UCC,
+        rc = MCA_PML_CALL(isend(tmpsend, msglen, MPI_BYTE, sendto, MCA_COLL_BASE_TAG_UCC,
                            MCA_PML_BASE_SEND_STANDARD, comm, &oob_req->reqs[0]));
-        MCA_PML_CALL(irecv(tmprecv, msglen, MPI_BYTE, recvfrom,
+	if (rc != OMPI_SUCCESS)
+		return rc;
```

Comment:
We have minimal style here in the Open MPI code base, but we do require 2 things:

1. 4 space tabs (no tab characters at all, please)
2. `{}` around all blocks, even 1-line blocks

Could you update your patch?

Thanks!

Reply:
Thanks for fixing.  I dismissed my "request changes" review, but I'll still defer to others to provide a formal review of this section of the code.